lib/gateway/index.js

```diff
@@ -20,12 +20,19 @@ module.exports = function ({ plugins, config } = {}) {
   const { httpServer, httpsServer } = bootstrap({ plugins, config });
 
   if (config.gatewayConfig.http && httpServer) {
+    // config.gatewayConfig.http.hostname = config.gatewayConfig.http.hostname || 'localhost';
```

Comment:
You can safely remove this line. We do not need to provide a default value since nodejs will handle that for us.